backend/authStrategies/index.ts

```
@@ -16,3 +18,45 @@
+    user = await User.findOne({ email: email })
+  }
+  const newUser: NewUser = {
+    fk: filter,
+    email: email,
+    name: userData.name,
+    access: (await getSettings()).defaultAccess
+  }
+  if (!user) {
+    user = new User(newUser)
+  } else {
+    delete newUser.access
```

Biome

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)